src/pathfinding_service/model/channel_view.py

```diff
@@ -42,25 +42,22 @@ def __post_init__(self) -> None:
         if self.capacity is None:
             self.capacity = self.deposit
 
-    # TODO: define another function update_deposit
+    def update_deposit(self, deposit: TokenAmount = TokenAmount(0)) -> None:
+
+        self.capacity = TokenAmount(self.capacity + deposit - self.deposit)
```

Comment:
I'd also check that `deposit` is bigger than `self.deposit`.

Reply:
what do we do if this is not the case?

Reply:
Then it wouldn't make sense to call the function